uktrade/data-hub-frontend
# Add Strategy form page for account management

## Description of change

Allow users to provide an overview of the strategic direction and long term goals for a company. 

## Test instructions

You should see a form with one input text area field and upon completion you should be redirected to the account management page.
URL is: companies/{companyId}/account-management/strategy/create

## Screenshots

### Before

_Add a screenshot_

### After

![Screenshot 2023-08-01 at 12 07 42](https://github.com/uktrade/data-hub-frontend/assets/14827050/4a579d24-4a78-47da-a6aa-583d27fc85fb)


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'

- [ x ] Has the branch been rebased to main?
- [ x ] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ x ] Manual compatibility testing (Browsers: Chrome, Firefox, Edge, Safari)
